app/models/labware_creators/pcr_cycles_binned_plate/csv_file_for_t_nano_seq.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
+require './lib/nested_validation'
+require 'csv'
+
+# Part of the Labware creator classes
+module LabwareCreators
+  require_dependency 'labware_creators/pcr_cycles_binned_plate'
+
+  #
+  # This version of the csv file is for Targeted NanoSeq.
+  #
+  class PcrCyclesBinnedPlate::CsvFileForTNanoSeq < PcrCyclesBinnedPlate::CsvFile
+    delegate :well_column,
+             :concentration_column,
+             :sanger_sample_id_column,
+             :supplier_sample_name_column,
+             :input_amount_available_column,
+             :input_amount_desired_column,
+             :sample_volume_column,
+             :diluent_volume_column,
+             :pcr_cycles_column,
+             :hyb_panel_column,
+             to: :well_details_header_row
+
+    def initialize_variables(file, config, parent_barcode)
+      @config = Utility::PcrCyclesForTNanoSeqCsvFileUploadConfig.new(config)
```

Comment:
This line could be extracted, either into a separate method to be overridden, or could pass in the relevant class. Then use the method from the parent class?